paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/viewmodel/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -37,15 +37,16 @@ internal class PaymentSheetPlaygroundViewModel(
     suspend fun prepareCheckout(
         customer: Repository.CheckoutCustomer,
         currency: Repository.CheckoutCurrency,
-        mode: Repository.CheckoutMode
+        mode: Repository.CheckoutMode,
+        setShippingAddress: Boolean = false
```

Comment:
seems this is only called with the value already set, can we just remove the default `false` here?

Reply:
Done